### PR TITLE
update nextjs-exception-middleware

### PR DESCRIPTION
**Pull request by itelo**

### CODE REVIEW

File: packages/nextlove/src/with-route-spec/index.ts [hunk 1]
```diff
@@ -51,12 +51,13 @@ export const createWithRouteSpec: CreateWithRouteSpecFunction = ((
...
-            process.env.CI ||
-              process.env.NODE_ENV === "development" ||
-              process.env.NODE_ENV === "test"
-          ),
+        getErrorContext: (req, error) => {
+          if (process.env.NODE_ENV === "production") {
+            return {}
+          }
+
+          return error
```
Comment (seveibar) [inline packages/nextlove/src/with-route-spec/index.ts:59]: @codetheweb should we serialize this into some kind of object?

@itelo is it possible to add a test for this?

Comment (codetheweb) [reply packages/nextlove/src/with-route-spec/index.ts:59]: This is what we do in a test on `nextjs-exception-middleware` and it seems to work, but yeah explicit serialization would be nice

Comment (itelo) [reply packages/nextlove/src/with-route-spec/index.ts:59]: looking at it

File: apps/example-todo-app/tests/api/todo/add.test.ts [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 import getTestServer from "tests/fixtures/get-test-server"
-import { v4 as uuidv4 } from "uuid"
 
 test("POST /todo/add", async (t) => {
   const { axios } = await getTestServer(t)
 
   const noAuthRes = await axios.post("/todo/add").catch((err) => err)
   t.is(noAuthRes.status, 401, "no auth")
 
+  const hasErrorStack = Boolean(noAuthRes.response.error.stack);
```
Comment (itelo) [inline apps/example-todo-app/tests/api/todo/add.test.ts:10]: @seveibar do you think this is enough?

Comment (seveibar) [reply apps/example-todo-app/tests/api/todo/add.test.ts:10]: yea that's pretty good!

**Review by seveibar — APPROVED** [review]

**Comment by seveibar** [comment]
actually @itelo can you see if there's also an error_code

File: apps/example-todo-app/tests/api/todo/add.test.ts (hunk 2 above)
Comment (seveibar) [reply apps/example-todo-app/tests/api/todo/add.test.ts:10]: let's check for an error code for an explicit exception

Comment (seveibar) [reply apps/example-todo-app/tests/api/todo/add.test.ts:10]: so if i throw `BadRequestException({ type: "my_error_code", message: "hello world" })`

we should see in our response
`{ error: { error_code, message } }`

Comment (itelo) [reply apps/example-todo-app/tests/api/todo/add.test.ts:10]: @seveibar Actually, we can see the error object even without the error handling; the only new data to the error object is the stack object

**Pull request merged**